### ISSUE TITLE
video.js problem

### ISSUE THREAD

**Issue by kvidek** [sachinchoolur/lightGallery#231]
Play/Pause not working on click on video. Also, cannot click on Play/Pause button at all.
Video JS plugin seems to work OK on its own, the problem is only present when video is opened in lightgallery.
HTML and JS are OK.

example:
http://nfgslice.deghq.com/?c=pages/who-are-we/index

**Comment by kvidek** [comment]
The issue seems to be present only when one video is in gallery. If there is more than one video - everything works fine. Any ideas?

**Comment by sachinchoolur** [comment]
Hi,
Thank you for pointing this out. It is a bug.
For current use change [lg-video.js](https://github.com/sachinchoolur/lightGallery/blob/master/src/js/lg-video.js#L57) line no 57 to 
``` javascript
if ($el.find('.lg-object').hasClass('lg-has-poster') && $el.find('.lg-object').is(':visible')) { 
```
I have been quite busy lately i will update it once i get time..

**Comment by kvidek** [comment]
Thank you very much.

**Comment by sachinchoolur** [comment]
The issue has been fixed 4ff464c0b702cb9cfede8e729d8acdec829655aa
